charts/policy-reporter/templates/deployment.yaml

```diff
@@ -58,13 +58,21 @@ spec:
               containerPort: 8080
               protocol: TCP
           livenessProbe:
+          {{- if .Values.livenessProbe }}
+            {{- toYaml .Values.livenessProbe | nindent 12 }}
+          {{- else }}
```

Comment:
Hey thanks for the contribution, can I ask what the intention of this override is? Could you add an empty livenessProbe/readinessProbe variable to the values.yaml. To see that it is possible to overwrite this configurations